rosidl_cmake/cmake/rosidl_cmake_export_typesupport_libraries-extras.cmake.in

```diff
@@ -0,0 +1,44 @@
+# generated from rosidl_cmake/cmake/template/rosidl_cmake_export_typesupport_libraries.cmake.in
+
+set(_exported_typesupport_libraries "@_ROSIDL_CMAKE_EXPORT_TYPESUPPORT_LIBRARIES@")
+
+# populate @PROJECT_NAME@_LIBRARIES_<suffix>
+if(NOT _exported_typesupport_libraries STREQUAL "")
+  # loop over typesupport libraries
+  foreach(_tuple ${_exported_typesupport_libraries})
+    string(REPLACE ":" ";" _tuple "${_tuple}")
+    list(GET _tuple 0 _suffix)
+    list(GET _tuple 1 _library)
+
+    if(NOT IS_ABSOLUTE "${_library}")
+      # search for library target relative to this CMake file
+      set(_lib "NOTFOUND")
+      find_library(
+        _lib NAMES "${_library}"
+        PATHS "${@PROJECT_NAME@_DIR}/../../../lib"
+        NO_DEFAULT_PATH NO_CMAKE_FIND_ROOT_PATH
+      )
+
+      if(NOT _lib)
+        # warn about not existing library and ignore it
+        message(FATAL_ERROR "Package '@PROJECT_NAME@' exports the typesupport library '${_library}' which couldn't be found")
```

Comment:
Should this be `STATUS` or `WARNING` or something else? Otherwise you're not going to "ignore it" as the comment says. It will stop here. I'm not sure what the behavior should be in this context.

Reply:
Hm, this is the same as in `ament_export_libraries`: https://github.com/ament/ament_cmake/blob/409ee0403d8d1eba0dc5a58cf52dcec5eadc91d2/ament_cmake_export_libraries/cmake/ament_cmake_export_libraries-extras.cmake.in#L47-L48

I would rather not change the behavior. So I just updated the comment to match the actual behavior: https://github.com/ros2/rosidl/pull/284/commits/03db54f021a1579e08b943731f4ba3996e050624#diff-d8035aae44630fe14ea78f925e0f4effR23